msteams-platform/overview-solution.md

```diff
@@ -0,0 +1,142 @@
+
+## The user story
+
+You've had a view of Teams offerings. You can now map them to user needs. Let’s revisit the scenario.
+
+The developer from Tours and Travel agency wants to build an app for their users, the travelers. The app should:
+
+- Check and send the forecast to travelers registered with the travel agency.
+- Notify the users a day before the departure date so they can plan.
+
+Collate and map requirements to Teams features:
+
+| User app needs | Check forecast | Notification before travel | Registered user |
+| --- |:---:|:---:|:---:|
+| **Capability** | Bot | &nbsp; | &nbsp; |
+| **Integration** | &nbsp; | &nbsp; | Microsoft Graph; Weather API |
+| **Scope** | &nbsp; | Personal app | &nbsp; |
+| **Integration point** | &nbsp; | Chat | &nbsp; |
+|
+
+**Teams app solution**: A Teams *personal chat bot* app that checks and *sends forecast notification* to *registered users* before their travel date.
+
+:::image type="content" source="../msteams-platform/assets/images/overview/developer-scenario-solution.png" alt-text="A developer at a travel agency builds a bot for Microsoft Teams that sends weather forecast to customers so that they can plan ahead their traveling dates" border="false":::
+
+Teams offers these and many more capabilities to bring your users a feature-rich app solution. To develop this app:
+
+1. Create a personal chat bot app.
+1. Integrate with an external weather forecast API to connect and request forecast for specific date and location.
+1. Integrate with Microsoft Graph for registered users.
+1. Check and send forecast details based on user's travel date and travel location.
+
+## Choose what suits you
+
+You can build a Teams app as per your app's requirements. Based on factors, such as business needs, development environment, domain knowledge, select the environment and tools you want to build your app.
+
+A Teams app offers you the flexibility of choosing your build environment. It includes tools, framework, and languages to approach your app development.
+
+:::image type="content" source="../msteams-platform/assets/images/overview/tools-of-your-choice.png" alt-text="Business need app" border="false":::
+
+Build your Teams app in the environment that works for your particular requirements. You can even select a combination.
+
+For example: You can use Teams Toolkit to build an app with JavaScript and host it on a SharePoint site.
+
+## Teams Collaborative Platform
+
+A Teams app brings your users the advantages of a collaborative workspace.
+
+As a platform for building apps, Teams offers the full range of apps and toolkits. Teams platform supports you at every stage from planning your app to distributing it.
+
+:::image type="content" source="../msteams-platform/assets/images/overview/teams-overview.png" alt-text="Illustration of Teams app building framework" border="true":::
```

Comment:
Let's seek help to get this image updated

Reply:
Replaced the diagram